gasometer/src/lib.rs

```diff
@@ -1,10 +1,26 @@
 //! EVM gasometer.
 
 #![deny(warnings)]
-#![forbid(unsafe_code, unused_variables, unused_imports)]
```

Comment:
Can you add back `unused_improts` warning here?

Reply:
It prevented to use `environmental` macro :/ Tried to allow it locally for the macro call but Rust complained.